swagger.yaml

```diff
@@ -8,6 +8,59 @@ security:
   - bearerAuth: []
 
 paths:
+  "/users/{userId}/vacation":
+    put:
+      operationId: updateUserVacation
+      summary: Update user vacation days
+      description: Update total and remaining vacation days for a specific user by year.
+      tags:
+        - Update vacation
```

Comment:
I think this should have the Users tag, so it is grouped with the other "Users" endpoints, it should also be moved in the spec to be with the other users endpoints e.g. below /users